src/mlpack/core/math/random.hpp

```diff
@@ -39,7 +39,7 @@ extern MLPACK_EXPORT std::normal_distribution<> randNormalDist;
  */
 inline void RandomSeed(const size_t seed)
 {
-  #if (BINDING_TYPE != BINDING_TYPE_TEST)
+  #if BINDING_TYPE == BINDING_TYPE_UNKNOWN || BINDING_TYPE != BINDING_TYPE_TEST
```

Comment:
I think it's probably better to use `defined(BINDING_TYPE)` instead of `BINDING_TYPE == BINDING_TYPE_UNKNOWN`.